static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -349,6 +349,24 @@ $block-height: 58px;
     }
 }
 
+// reserve space for comment count on commentable card
+.fc-item--is-commentable:not(.fc-item--type-comment) .fc-item__meta {
+    min-height: 17px;
+    min-width: 40px;
```

Comment:
could these be `em`s or `ch`s? numbers feel a bit magic?

Reply:
wonder if 3/4 `&nbsp;`s in the target element would work at all?

Reply:
If this is to prevent layout thrashing, is a hardcoded `min-width` necessary?

Reply:
The `min-width` is required to prevent the standfirst's width decreasing when the comment container is injected adjacent to it. Will look at @sndrs `&nbsp;` suggestion.

Reply:
@GHaberis yeah not sure if it's stupid or would actually cover all bases?

Reply:
@sndrs you'd still need to account of the comment icon's width of 14px. The numbers aren't totally magic `17px` is the comment-icons height plus it's margin-top (14px + 3px). `40px` is the the width of `.fc-trail__count--commentcount` with a 3 character comment-count.

Reply:
Perhaps a comment on these properties would be enough?